oceandb_elasticsearch_driver/instance.py

```diff
@@ -12,15 +12,26 @@ def get_database_instance(config_file=None):
 class ElasticsearchInstance(object):
     def __init__(self, config=None):
         host = get_value('db.hostname', 'DB_HOSTNAME', 'localhost', config)
-        port = int(get_value('db.port', 'DB_PORT', 27017, config))
+        port = int(get_value('db.port', 'DB_PORT', 9200, config))
         username = get_value('db.username', 'DB_USERNAME', None, config)
         password = get_value('db.password', 'DB_PASSWORD', None, config)
         index = get_value('db.index', 'DB_INDEX', 'oceandb', config)
+        ssl = get_value('db.ssl', 'DB_SSL', False, config)
+        verify_certs = get_value('db.verifyCerts', 'DB_VERIFY_CERTS', False, config)
+        ca_certs = get_value('db.caCertPath', 'DB_CA_CERTS', None, config)
+        client_key = get_value('db.clientKey', 'DB_CLIENT_KEY', None, config)
+        client_cert = get_value('db.clientCertPath', 'DB_CLIENT_CERT', None, config)
+
         self._index = index
         self._es = Elasticsearch(
             [host],
             http_auth=(username, password),
```

Comment:
I was trying to run it, and it seems that if the username and password are `None` you can not pass here. You should check first if you have an user and password, and try to login later of this with the user and password.